### PR TITLE
Fix d ts updates

### PR DESCRIPTION
**Pull request by brauliodiez**
Fixed sample 14 test components.

We have still the redux-mock-store manual d.ts, but submitted a new PR to the redux-mock-store in order to get updated the d.ts on their project

[https://github.com/arnaudbenard/redux-mock-store/issues/52](https://github.com/arnaudbenard/redux-mock-store/issues/52)

[https://github.com/arnaudbenard/redux-mock-store/pull/51](https://github.com/arnaudbenard/redux-mock-store/pull/51)

On the other hand redux-thunk definition is outdated, needed a manual one

Once approved this changes should be propagated to other samples using this (15? )

### CODE REVIEW

File: 14 TestComponent/src/components/common/spec/spinner.container.spec.tsx [hunk 1]
```diff
@@ -18,7 +20,15 @@ describe('Spinner container component', () =>{
...
         });
 
-        let httpInitializeDispatcherActionStub = sinon.stub(httpActions, 'httpInitializeDispatcher');
+        let httpInitializeDispatcherActionStub = sinon.stub(httpActions,
+                                               'httpInitializeDispatcher',
+                                               () => {
+                                                 return {
+                                                   type: 'dummy'
+                                                 }
+                                               }
```
Comment (brauliodiez) [inline 14 TestComponent/src/components/common/spec/spinner.container.spec.tsx]: Update on new Redux version, action types cannot be "undefined"

File: 10 AsyncCalls_Redux_Thunk/tsd.json [hunk 2]
```diff
@@ -29,15 +29,9 @@
     "object-assign/object-assign.d.ts": {
       "commit": "4ba57e605bd71bf7248de65aac0b4f50e3eb1c9e"
     },
-    "redux/redux.d.ts": {
```
Comment (Nasdan) [inline 10 AsyncCalls_Redux_Thunk/tsd.json]: We still have errors (this time with react-redux.d.ts) due to use redux.d.ts from redux js lib (index.d.ts)

```
ERROR in c:\Dev\Lemoncode\React-Typescript-Samples\10 AsyncCalls_Redux_Thunk\typings\react-redux\react-redux.d.ts
(34,16): error TS2314: Generic type 'Dispatch<S>' requires 1 type argument(s).

ERROR in c:\Dev\Lemoncode\React-Typescript-Samples\10 AsyncCalls_Redux_Thunk\typings\react-redux\react-redux.d.ts
(38,21): error TS2314: Generic type 'ActionCreator<A>' requires 1 type argument(s).

ERROR in c:\Dev\Lemoncode\React-Typescript-Samples\10 AsyncCalls_Redux_Thunk\typings\react-redux\react-redux.d.ts
(60,13): error TS2314: Generic type 'Store<S>' requires 1 type argument(s).

ERROR in c:\Dev\Lemoncode\React-Typescript-Samples\10 AsyncCalls_Redux_Thunk\typings\react-redux\react-redux.d.ts
(7,1): error TS6053: File 'c:/Dev/Lemoncode/React-Typescript-Samples/10 AsyncCalls_Redux_Thunk/typings/redux/redux.d.ts' not found.
```

I've tried to upload screenshot but github fails on upload image

File: 10 AsyncCalls_Redux_Thunk/tsd.json [hunk 3]
```diff
@@ -29,15 +29,9 @@
     "object-assign/object-assign.d.ts": {
```
Comment (Nasdan) [inline 10 AsyncCalls_Redux_Thunk/tsd.json]: You update tsd.json from sample 10 but not in sample 14

**Comment by Nasdan** [comment]
This PR is ready.

**Comment by Nasdan** [comment]
We are using redux-mock-store index.d.ts from original library. So, we don't need manual typings

**Pull request merged**